networkx/algorithms/components/tests/test_semiconnected.py

```diff
@@ -53,3 +53,10 @@ def test_alternating_path(self):
             chain.from_iterable([(i, i - 1), (i, i + 1)] for i in range(0, 100, 2))
         )
         assert not nx.is_semiconnected(G)
+
+    def test_topo_order(self):
+        G = nx.path_graph(100, create_using=nx.DiGraph())
```

Comment:
Just a minor nit to keep an empty graph from being created. More style than impact.
```suggestion
        G = nx.path_graph(100, create_using=nx.DiGraph)
```